Tests/test_file_tiff_metadata.py

```diff
@@ -136,14 +136,6 @@ def test_write_metadata(self):
         original = img.tag_v2.named()
         reloaded = loaded.tag_v2.named()
 
-        for k, v in original.items():
-            if isinstance(v, IFDRational):
-                original[k] = IFDRational(*_limit_rational(v, 2 ** 31))
-            elif isinstance(v, tuple) and isinstance(v[0], IFDRational):
-                original[k] = tuple(
-                    IFDRational(*_limit_rational(elt, 2 ** 31)) for elt in v
-                )
-
```

Comment:
Why remove this part of the test?

Reply:
It seems disfunctional and wrong. 
The fact that Pillow used to incorrectly limit unsigned rationals to `2**31`, does not need to be validated. TIFF allows for rationals to be up to 2**32. If original file contained bigger (but still valid) value, then it has to be the same when Pillow saves it.